eZ/Publish/Core/Repository/Validator/VersionValidator.php

```diff
@@ -68,7 +68,7 @@ public function validate(
             );
 
             foreach ($languageCodes as $languageCode) {
-                $fieldValue = $content->getField($fieldDefinition->identifier)->value ?? $fieldDefinition->defaultValue;
+                $fieldValue = $content->getField($fieldDefinition->identifier, $languageCode)->value ?? $fieldDefinition->defaultValue;
```

Comment:
Unfortunately, this change introduces a regression, ref: https://github.com/ezsystems/ezplatform-kernel/pull/246.

Reply:
Well, to be more precise: if the above steps don't lead to the issue anymore concerning the scope of this PR, we should be good to go.

Reply:
I tested this ticket with steps from your comment, and the problem does not occur anymore